src/lib/dnssd/minimal_mdns/AddressPolicy.cpp

```diff
@@ -25,9 +25,22 @@ namespace {
 AddressPolicy * gAddressPolicy = nullptr;
 } // namespace
 
+// This will be resolved at link time if a default policy is set
+#ifndef CHIP_MINMDNS_NONE_POLICY
+AddressPolicy * GetDefaultAddressPolicy();
```

Comment:
this should go in some header rather than declared here and then in other per-policy headers in a repeated manner.

We want to ensure a single declaration placement. Probably in `AddressPolicy.h` . Full documentation on usage should be described there.